react/components/ExtensionPoint/ComponentLoader.tsx

```diff
@@ -66,10 +66,16 @@ const ComponentLoader: FunctionComponent<Props> = props => {
     treePath,
     props: componentProps,
     hydration,
-    runtime,
   } = props
 
-  // Each prop that receives a slot should always be PascalCased.
+  /**
+   * Props that function as Slots are ALWAYS CamelCased.
+   * It is OK to not include componentProps in the dependency array
```

Comment:
Coudn't this be done by something above this component?

Reply:
I couldn't make it work :(
Tried to use `useMemo` on `generateSlots` function without any success.